DDD.Functions/EloVotingGetPair.cs

```diff
@@ -75,10 +76,21 @@ EloVotingConfig eloVoting
             var sessionIdsLoader = new Lazy<Task<List<string>>>(async () => await LoadSessionIds(submissions, conference));
 
             // get the voting session id from the cookie from the user's browser, creating a new one if it doesn't exist
-            var userSessionId = string.IsNullOrEmpty(
-                req.Cookies[submissions.UserVotingSessionCookieName ?? DefaultCookieName])
-                ? Guid.NewGuid().ToString()
-                : req.Cookies[submissions.UserVotingSessionCookieName ?? DefaultCookieName];
+            var headerKey = string.IsNullOrEmpty(submissions.UserVotingSessionHeaderName)
+                ? DefaultSessionIdHeaderName
+                : submissions.UserVotingSessionHeaderName;
+
+            string userSessionId = null;
+            if (req.Headers.ContainsKey(headerKey))
+            {
+                userSessionId = req.Headers[headerKey];
+            }
+            
+            if (string.IsNullOrEmpty(userSessionId))
+            {
+                log.LogWarning("Could not get session identifier from header {headerName}.", headerKey);
```

Comment:
Guess this means you need the front-end change before you can merge?

Reply:
I usually test the functions with cURL/Postman, so I can test manually :-)

Reply:
Sure, but if we deploy this version, then the website stops working because it doesn't (yet) include the header